base_export_async/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     'summary': """
         Asynchronous export with job queue
         """,
-    'version': '12.0.1.1.0',
+    'version': '12.0.1.1.1',
```

Comment:
please do not change module version, this is done by the bot. Also, this is a new feature and deserves a + 0.1.0 :wink:

Reply:
Thank you for your information, I have Updated manifest file.